add equipo2 module views and change navbar

Mike5801 opened the pull request:
Views for app modules: QR, RBAC, Feedback and FAQ/Support

Mike5801 committed:
add event view

aruizgarciarojas committed:
add event view (#1)

This commit adds the "selected event" view in folder layout and also
delete unnecessary test.txt files in all folders

Mike5801 committed:
Delete .idea directory

Commit:
Delete .idea directory (#3)

Delete .idea folder to prevent merge conflicts

Mike5801 committed:
update git ignore

add .idea to prevent merge conflicys

Commit:
update git ignore (#4)

add .idea to prevent merge conflicts

Commit:
Merge branch 'Equipo2' into develop

Mike5801 committed:
add navbar to layout and equipo1's views (#5)

Adds navbar component and equipo1's views

Commit:
Merge branch 'Mike5801:develop' into develop

Commit:
add changes of develop to Equipo2 (#7)

add changes of develop to branch Equipo2

Mike5801 committed:
Merge pull request #16 from Mike5801/develop

add folders domain and models

Renato549 committed:
add regiter, login. scanner

Commit:
add regiter, login. scanner (#10)

add activity register log in and qr 's scanner

halimabraham committed:
add contact, faq, support, failures and verification layouts

Commit:
add contact, faq, support, failures and verification layouts (#11)

Mike5801 committed:
add Equipo2 Module Views and correct navbar

Commit:
add Equipo2 Module Views and correct navbar (#12)

Add Views and change navbar

Mike5801 committed:
Merge pull request #19 from MiguelWTangF/Equipo2

add equipo 2 module views

Mike5801 committed:
Merge branch 'develop' into Equipo2

Commit:
Merge branch 'Mike5801:develop' into develop

Commit:
Develop (#13)

Commit:
Merge branch 'Mike5801:Equipo2' into Equipo2

Commit:
Merge branch 'Mike5801:develop' into develop

Commit:
add Repository with Room implementation (#14)

add domain and model folders with repository and room implementation

MandukoGR commented on app/src/main/res/layout/activity_actividades_organizador.xml:
```diff
@@ -94,7 +94,6 @@
                     android:layout_margin="32dp"/>
-
             </LinearLayout>
 
```
Chequen si al correr la app se ve bien, nosotros tuvimos problemas con los Linear Layout

MiguelWTangF replied:
Se ve con la imagen del evento y el titulo bien, pero el recycler view se ve como una lista sin nada y sin estil, mando captura.
![image](https://user-images.githubusercontent.com/113156580/189718654-35a441ce-25cd-4bc1-85c8-f6ee7dab72f4.png)

MandukoGR replied:
está bien aún no lo ponemos

MandukoGR commented on app/src/main/res/layout/navbar.xml:
```diff
@@ -2,157 +2,151 @@
     android:layout_gravity="bottom"
     >
-
     <androidx.constraintlayout.widget.ConstraintLayout
```
No debería de estar solo en un constraint?

MiguelWTangF replied:
Se necesita el constraint layout exterior para que el interno esté ajustado al tamaño fijo y al importarlo siempre tenga las mismas dimensiones. Digamos que el constraint exterior lo que hace es la base que se adapta al contenedor, y el segundo constraint es el contenedor = tamaño real del navbar. Si lo quitamos no se va ajustar el tamaño cuando le asignemos un id al include (Este id tiene que ser el mismo  que el id del constraint principal del navbar.xml en el caso de que lo quitemos), y se va a ajustar al límite que utilicen los objetos en este caso los íconos por el wrap content que tiene, y si lo defines de un tamaño en específico (80dp) antes de importarlo y asignarle un id se regresa a un wrap content de los contenedores (iconos y texto), se lo tendrías que asignar externamente en el include en lugar de hacer el wrap content y match parent que creo que es mejor

Mike5801 committed:
add connection to Firebase, Firestore and Firebase Auth

Commit:
add connection to Firebase, Firestore and Firebase Auth (#15)

Add imports and dependencies for connection to Firebase, Firestore and
Firebase Auth

Mike5801 committed:
Merge pull request #22 from MiguelWTangF/Equipo2

Add connection to firebase, firestore and firebase auth
